website/server/libs/payments.js

```diff
@@ -128,13 +137,26 @@ api.createSubscription = async function createSubscription (data) {
     }
   }
 
-  await data.user.save();
+  if (group) {
+    await group.save();
+  } else {
+    await data.user.save();
+  }
+
   if (data.gift) await data.gift.member.save();
 };
 
 // Sets their subscription to be cancelled later
 api.cancelSubscription = async function cancelSubscription (data) {
   let plan = data.user.purchased.plan;
+  let group;
```

Comment:
Actually, I am not seeing where the entire group is passed. I haven't updated the code, but I remember your link showing a place... now I can't find it.